catroid/src/main/res/values-de/strings.xml

```diff
@@ -1689,7 +1689,7 @@ Die vorangegangene, veraltete Version unserer Datenschutzrichtlinie (2.0) finden
     <string name="formula_editor_logic_comparison">Vergleichsoperatoren</string>
     <string name="formula_editor_object_general">Allgemeine Eigenschaften</string>
     <string name="formula_editor_object_movement">Physikalische Eigenschaften</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations